jovian/jovian_nb_ext/main.js

```diff
@@ -609,6 +710,155 @@ define([
       jvn_params_modal.modal("show");
     };
 
+    //just save new default parameters
+    const saveParams = function() {
+      /**
+       * Initializes a dialog modal triggered by a dropdown button on the toolbar
+       *
+       * Body: formParamsUI()
+       *
+       */
+      const jvn_params_modal = dialog.modal({
+        show: false,
+        title: "Set up Parameters to Jovian",
+        body: formParamsUI,
+        notebook: Jupyter.notebook,
+        keyboard_manager: Jupyter.notebook.keyboard_manager,
+        buttons: {
+          Set: {
+            id: "save_params_button",
+            class: "btn-primary",
+            click: function() {
+              storeParamsInPython();
+            }
+          }
+        },
+        open: function() {
+          getParams().then(jvn_params => {
+            if (jvn_params == null) {
+              $("#nb_filename_box").val(
+                Jupyter.notebook.notebook_name.replace(".ipynb", "")
+              );
+              $($("input[name=secret_opt")[1]).prop("checked", true);
```

Comment:
These changes were merged and changed. Update the branch and resolve conflicts before next commit

Reply:
Pull the latest master and merge, you might have older master branch when you merged.